tests/test_engine.py

```diff
@@ -47,6 +47,18 @@ def load_rulebook(rules_file):
     return ruleset_queues, event_log
 
 
+async def get_queue_item(queue, timeout=0.5, times=1):
+    for _ in range(times):
+        try:
+            item = queue.get_nowait()
+        except asyncio.QueueEmpty:
+            await asyncio.sleep(timeout)
+            continue
+        if item:
```

Comment:
What if the item returned evaluates to False? Don't you wanted it?

Reply:
@Alex-Izquierdo Its always a dict coming into the event_log queue, it is never an individual scalar value